slack/send/README.md

```diff
@@ -2,11 +2,23 @@
 
 ## Inputs
 <!--inputs-->
-| Name         | Description                                       | Required | Default |
-|--------------|---------------------------------------------------|----------|---------|
-| `channel-id` |                                                   | `true`   | ` `     |
-| `bot-token`  | Specify the slack bot token.                      | `true`   | ` `     |
-| `message`    | Specify the message to be sent (markdown format). | `true`   | ` `     |
+| Name         | Description                                                                                                                                                                                                                                                                                                                                    | Required | Default |
+|--------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|---------|
+| `bot-token`  | Specify the slack bot token.                                                                                                                                                                                                                                                                                                                   | `true`   | ` `     |
+| `channel-id` |                                                                                                                                                                                                                                                                                                                                                | `true`   | ` `     |
+| `message`    | Slack message on Markdown format. Multiline messages must be escaped using URL encoding.
+https://github.com/orgs/community/discussions/26288
```

Comment:
as far as i see the automation does not support a multline approach, though GitHub input descriptions allow so.

Reply:
@reakaleek , what are your thoughts regarding multiline descriptions and auto-docs generation?

Reply:
Oh. Good point. Let me take a quick look at this.

Reply:
Thanks Jan, btw, there is no urgency, those changes can be supported in a follow-up as they are cosmetic IMO

Reply:
I found a simple fix for this.. but needs a new release in reakaleek/gh-action-readme. I only need to replace `\n` with `<br>` and the table will be look as expected

Reply:
https://github.com/elastic/oblt-actions/pull/32 should fix it